logstreams/src/main/java/io/zeebe/logstreams/state/StateSnapshotController.java

```diff
@@ -119,7 +115,7 @@ public long recover() throws Exception {
         // open database to verify that the snapshot is recoverable
         openDb();
         LOG.debug("Recovered state from snapshot '{}'", snapshot);
-        lowerBoundSnapshotPosition = snapshot.getPosition();
+        break;
```

Comment:
:see_no_evil: This is often really hard to spot

Reply:
Hm, the other option is to use a boolean flag, I guess - I'm usually fine with a break but we can use that instead.

Reply:
Leave it for now - we will probably rewrite this method in the next issues, because we remove multiple snapshots.